Lib/test/test_os.py

```diff
@@ -2077,6 +2077,13 @@ def test_file_link(self):
         self.assertTrue(os.path.islink(self.filelink))
         self.check_stat(self.filelink, self.filelink_target)
 
+    @unittest.skipUnless(hasattr(os, 'readlink'), 'needs os.readlink()')
+    def test_readlink_pathlike(self):
```

Comment:
Seems there are no tests for `readlink()` with strings and bytes. Would be nice to test them too.

See also `OSErrorTests.test_oserror_filename`. Perhaps it can be simplified.

Reply:
> Seems there are no tests for readlink() with strings and bytes. Would be nice to test them too.

Ok, I will create a separate `ReadlinkTests` test case and move `test_readlink_pathlike` there as well.